config/default/traefik.yaml

```diff
@@ -0,0 +1,25 @@
+globalArguments:
+  - "--global.checknewversion"
+logs:
+  general:
+    format: json
+  access:
+    enabled: true
+    format: json
+providers:
+  kubernetesIngress:
+    publishedService:
+      enabled: true
```

Comment:
```suggestion
  kubernetesIngress:
    publishedService:
      enabled: true
  kubernetesCRD:
    enabled: true
```

Let's enable the 2 CRD implementation to allow defining the custom resource that provides way more features (I'm thinking about the Nginx advances rules that might not be provided by Traefik's default Ingress implementation).

I also see this as a nice help to ensure we don't mess up between Nginx: A Traefik-enabled application would have `HTTPRoute` object while a Nginx-enabled would have `IngressRule`. At least it would be clear, because Ingress Rule's portability is a lie: better to assume that IF we accept such a change, then we don't mix up older and newer rules.

Reply:
kubernetesCRD is enabled by default.

Reply:
👍 Based on the CI error message, it sounds right. I wasn't sure about the map's merging behavior in the `values.yaml`. Thanks for the precision.